chapters/learning.md

```diff
@@ -4,11 +4,11 @@ There are many basic ideas from the science of learning that most people do not
+To build efficiently on prior knowledge, learners need to do deliberate practice <ericsson1993>. The need for practice stems from the idea that most human knowledge and abilities are acquired and improved through repeated exercising of a skill, targeted feedback on what aspects of that skill has mastered and needs to improve, and sufficient motivation to persist with practice. For example, if you're teaching the concept of privacy, it's not enough to just explain the idea, you need students to practice explaining the idea themselves, you need to give targeted feedback on their explanations of privacy, and you need to keep them motivated to improve on their explanations. Or, if you're teaching a student a new programming language, it's not enough to explain the concept of the language's function calls, you need to give examples, have students practice reading and explaining how the examples execute, and give targeted feedback on their understanding. An important implication of the need for deliberate practice is that most people can learn most things with sufficient practice, but not just any kind of practice will suffice. This body of evidence is pretty damning for most schools, including higher education, which rarely provides targeted feedback, and often does not even make time for repetition of a skill. A teacher's job is to structure an environment to promote deliberate practice for the learning objectives of a class, but students are often left to structure their own practice.
```

Comment:
I added "do" before deliberate practice here, not sure why diff is failing to highlight that there. It could also be "need deliberate practice".

Reply:
Oh, also changed school to schools. Could also be schooling?